tox.ini

```diff
@@ -51,7 +51,7 @@ deps =
     -rrequirements-jammy.txt
 description = Run tests with pytest
 labels =
-    py3.{10,11}: tests
```

Comment:
We might as well remove 3.11 here

Reply:
I would like to keep it until 24.04 released